### ISSUE TITLE
Support for setting commit

### ISSUE THREAD

**Issue by danilofuchs** [getsentry/sentry-dart-plugin#19]
Release tracking gets even better with Commit Hash info when the GitHub source repo is integrated with Sentry.

The `set-commits` command would be useful if integrated

**Comment by marandaneto** [comment]
@danilofuchs indeed, would you like to contribute with a PR? this plugin was a fun hack week project, we don't have anyone working on it actively (other than bug fixing), but happy to guide contributors :)

**Comment by marandaneto** [comment]
Consider adding support for React native as well, I will create an issue for it.

**Comment by marandaneto** [comment]
@vaind would you like to give it a stab at https://github.com/getsentry/sentry-dart-plugin/issues/43 ?
Meta issue https://github.com/getsentry/team-mobile/issues/33
Example using `--auto` https://github.com/sentry-demos/android/blob/8c9d690293d3ebe76e760bfb0718da883885ebd6/Makefile#L10-L18
Sentry CLI docs https://docs.sentry.io/product/cli/releases/#sentry-cli-commit-integration

I believe that the plugin should offer an `auto` configuration, if it is disabled and a commit is specified, use the specified commit.

**Comment by vaind** [comment]
> @vaind would you like to give it a stab at #43 ? Meta issue [getsentry/team-mobile#33]

yes, that's actually the issue that brought me here, but I had to do some yak shaving first so that I could be confident my changes won't break other things.

**Comment by marandaneto** [comment]
@vaind wondering if this should be considered:

> Maybe using --ignore-missing by default as well https://docs.sentry.io/product/cli/releases/#dealing-with-missing-commits

Since you've not replied nor implemented, was there a reason to not do this by default?
It's not super important, just trying to narrow down the problem [here](https://github.com/getsentry/sentry-dart-plugin/issues/68).

**Comment by vaind** [comment]
Right, I guess I've forgotten to reply here. I guess the same question has already been considered elsewhere and the decision was to make it an opt-in in the CLI. I think It would be good to check with the original developers who may have had good reasons to do so - potential issues we're not even considering here.
